make/configure.py

```diff
@@ -1238,6 +1238,10 @@ def createCLI( cross = None ):
     grp.add_argument( '--enable-x265', dest="enable_x265", default=True, action='store_true', help=(( 'enable %s' %h ) if h != argparse.SUPPRESS else h) )
     grp.add_argument( '--disable-x265', dest="enable_x265", action='store_false', help=(( 'disable %s' %h ) if h != argparse.SUPPRESS else h) )
 
+    h = IfHost( 'x265 NUMA support', '*-*-linux', none=argparse.SUPPRESS ).value
+    grp.add_argument( '--enable-numa', dest="enable_numa", default=True, action='store_true', help=(( 'enable %s' %h ) if h != argparse.SUPPRESS else h) )
```

Comment:
Default for Linux and Windows:

`default=(host.match( '*-*-linux*' ) or (cross is not None and 'mingw' in cross))`

Default for Linux only:

`default=(host.match( '*-*-linux*' ) and cross is None)`

Reply:
I've taken the more difficult approach of fixing build vs host notation.  So of coarse much more testing will now be needed on all platforms :tongue:

Reply:
Might still want to make the default platform-specific here.

Reply:
Might still want to make the default platform-specific here.

Reply:
Fixed